src/routes/PostDetail/PostDetail.connector.js

```diff
@@ -49,22 +48,22 @@ export function mapDispatchToProps (dispatch, props) {
     onClose: () => dispatch(push(closeLocation)),
     joinProject: () => dispatch(joinProject(postId)),
     leaveProject: () => dispatch(leaveProject(postId)),
-    voteOnPost: (myVote) => dispatch(voteOnPost(postId, myVote)),
     processStripeToken: (postId, token, amount) => dispatch(processStripeToken(postId, token, amount)),
     respondToEvent: response => dispatch(respondToEvent(postId, response))
   }
 }
 
-export function mergeProps (stateProps, dispatchProps, ownProps) {
-  const { post } = stateProps
+// export function mergeProps (stateProps, dispatchProps, ownProps) {
```

Comment:
I don't think any of this is needed but I'm not completely sure how to test. 

There are no props being merged in the 'mergeProps' function, sooo.... delete?